Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/ScaleOffsetColorTheme.cs

```diff
@@ -67,8 +67,8 @@ public override void Init(GameObject host, ThemeDefinition settings)
         {
             base.Init(host, settings);
             hostTransform = Host.transform;
-            startPosition = hostTransform.localPosition;
-            startScale = hostTransform.localScale;
+            originalPosition = hostTransform.localPosition;
```

Comment:
ScaleOffsetColor theme is used quite a often in MRTK, it would be nice to add the "relative" property here as well.

Reply:
@Troy-Ferrell said it wasn't necessary on the ScaleOffsetColor :p
i can still add it but it would default to 'true'

Reply:
I don't think it's necessary because both Scale and Offset are purely relative in this theme and I was trying to remove obstacles to get the rest of the PR in. Feel free to add it though